### PR TITLE
Create a gcr.io image for Flunentd to Elasticsearch node level logging

### PR DESCRIPTION
**Pull request by satnam6502**

### CODE REVIEW

File: cluster/addons/fluentd-elasticsearch/fluentd-es-image/Makefile [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
 TAG = 1.3
 
 build:	
-	sudo docker build -t kubernetes/fluentd-elasticsearch:$(TAG) .
+	docker build -t gcr.io/google-containers/$(IMAGE):$(TAG) .
+	docker tag -f gcr.io/google-containers/$(IMAGE):$(TAG) gcr.io/google-containers/$(IMAGE):$(TAG)
 
 push:	
-	sudo docker push kubernetes/fluentd-elasticsearch:$(TAG)
+	gcloud preview docker push gcr.io/google-containers/$(IMAGE):$(TAG)
```
Comment (roberthbailey) [inline cluster/addons/fluentd-elasticsearch/fluentd-es-image/Makefile]: You should specify the project here so it doesn't need to be set in the environment: `gcloud --project=google-containers preview docker push ...`

Comment (jlowdermilk) [inline cluster/addons/fluentd-elasticsearch/fluentd-es-image/Makefile]: The project is determined from the image name/tag; it doesn't need to be passed as a flag. Per [documentation](https://cloud.google.com/tools/container-registry/#pushing_to_the_registry), in order to push to a gcr.io repository, the image has to be tagged `gcr.io/project-id/image-name:tag`. The repository location is `gcr.io/project-id`.

**Comment by satnam6502** [comment]
Ah, this has a redundant tag too?

**Comment by jlowdermilk** [comment]
lgtm

**Pull request merged**